components/org.wso2.carbon.identity.application.authenticator.adapter/src/main/java/org/wso2/carbon/identity/application/authenticator/adapter/util/AuthenticatedUserBuilder.java

```diff
@@ -214,16 +232,21 @@ private AbstractUserStoreManager resolveUserStoreManager(AuthenticatedUserData.U
             }
         } catch (UserStoreException e) {
             if (userStore != null && StringUtils.isNotBlank(userStore.getName())) {
-                throw new ActionExecutionResponseProcessorException(String.format("An error occurred while " +
-                        "retrieving the userStore manager for the given userStore domain: %s.", userStore.getName()),
-                        e);
+                String errorMessage = "An error occurred while retrieving the userStore manager for the given " +
+                        "userStore domain: " + userStore.getName();
+                logSuccessResponseExecutionErrorResult(new AuthenticationActionExecutionResult("userStore",
+                        Availability.AVAILABLE, Validity.INVALID, errorMessage));
+                throw new ActionExecutionResponseProcessorException(String.format(errorMessage), e);
             }
             throw new ActionExecutionResponseProcessorException("An error occurred while fetching the userStore " +
                     "manager for the default userStore domain.", e);
         }
         if (userStoreManager == null) {
-            throw new ActionExecutionResponseProcessorException(String.format("No userStore is found for the given " +
-                    "userStore domain name: %s.", userStore.getName()));
+            String errorMessage = "No userStore is found for the given userStore domain name: %s." +
+                    userStore.getName();
```

Comment:
This is of wrong use right. Looks like you have used a formatting wild card while appending the value.
Let's use the concatenation rather than going towards string formatting